app/views/rubygems/_gem_members.html.erb

```diff
@@ -4,6 +4,21 @@
     <div class="gem__users">
       <%= links_to_owners(rubygem) %>
     </div>
+
+    <% if  rubygem.owned_by?(current_user) %>
+      <% if rubygem.owners.without_mfa.present? %>
```

Comment:
.present? loads the entire relation (`SELECT * FROM USERS`). You don't actually want to do that here. Substitute `exists?` instead.